cfgmgr/teammgr.cpp

```diff
@@ -380,7 +381,15 @@ bool TeamMgr::addLag(const string &alias, int min_links, bool fallback)
     SWSS_LOG_INFO("Port channel %s teamd configuration: %s",
             alias.c_str(), conf.str().c_str());
 
-    cmd << TEAMD_CMD << " -r -t " << alias << " -c " << conf.str() << " -d";
+    string warmstart_flag = WarmStart::isWarmStart() ? " -w " : "";
+    const string dump_path = "/var/warmboot/teamd/";
+
+    cmd << TEAMD_CMD
+        << warmstart_flag
+        << " -r -t " << alias
+        << " -c " << conf.str()
+        << " -L " << dump_path
+        << " -d";
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
```

Comment:
>EXEC_WITH_ERROR_THROW [](start = 4, length = 21)

For old code: execve() will be better then system(), in case of user input attach.
For example, 'alias' may contain malicious content with blanks inside.